### PR TITLE
Adds heading alpha element.

### PR DESCRIPTION
**Pull request by gabrielpoca**

### CODE REVIEW

**Comment by gabrielpoca** [comment]
@Jvxtv review please.

File: client/components/heading/_heading.sass [hunk 1]
```diff
@@ -1,17 +1,45 @@
...
   .HeadingAlpha-extra
     display: inline-block
-    vertical-align: bottom
+    height: 100%
 
   .HeadingAlpha-title
-    font-size: 1.5rem
-    color: $blue
-    width: 70%
+    font-size: 2.2rem
```
Comment (joaojusto) [inline client/components/heading/_heading.sass:23]: What do you think of putting these font-sizes into variables so we can reuse them easily later?

Comment (gabrielpoca) [inline client/components/heading/_heading.sass:23]: Well, it's not that I don't agree, it's just that I don't see where in particular font-size can be reused. And using variables just for the sake of using them just makes things harder. We'll discuss this in person.

**Comment by joaojusto** [comment]
:thumbsup:

**Comment by joaojusto** [comment]
:thumbsup:

**Comment by joaojusto** [comment]
Ready to merge

**Pull request merged**